libqtile/layout/tile.py

```diff
@@ -155,14 +157,18 @@ def configure(self, client, screen_rect):
                 bc = self.border_focus
             else:
                 bc = self.border_normal
+            if not self.border_on_single and len(self.clients)==1:
+                border_width = 0
+            else:
+                border_width = self.border_width
             client.place(
                 x,
                 y,
                 w - border_width * 2,
                 h - border_width * 2,
                 border_width,
                 bc,
-                margin=self.margin,
+                margin = 0 if (not self.margin_on_single and len(self.clients)==1) else self.margin,
```

Comment:
Spaces are needed around `==` here and on line 160 - this will get caught by the CI which I have just enabled

Reply:
Dear developer,  I was careless and added space near both `==` and `=`, which lead to the previous failed action.  I just found a tool named autopep8 in the gentoo repository, which helped me spot the space problem.  I guess there would be no problem here this time= =